src/Aspire.Hosting.Azure/AzureConstructResource.cs

```diff
@@ -101,6 +101,23 @@ public static IResourceBuilder<AzureConstructResource> AddAzureConstruct(this ID
                       .WithManifestPublishingCallback(resource.WriteToManifest);
     }
 
+    /// <summary>
+    /// Configures the Azure construct resource.
+    /// </summary>
+    /// <typeparam name="T">Type of the CDK resource.</typeparam>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="configure">The configuration callback.</param>
+    /// <returns>The resource builder.</returns>
+    public static IResourceBuilder<T> ConfigureConstruct<T>(this IResourceBuilder<T> builder, Action<ResourceModuleConstruct> configure) 
```

Comment:
Should be marked experimental?

Reply:
The constructor isn't so I don't think so.